packages/vaex-core/vaex/dataframe.py

```diff
@@ -5747,6 +5753,41 @@ def export_fits(self, path, column_names=None, shuffle=False, selection=False, p
         import vaex.export
         vaex.export.export_fits(self, path, column_names, shuffle, selection, progress=progress, virtual=virtual, sort=sort, ascending=ascending)
 
+    @docsubst
+    def export_csv(self, path, virtual=False, selection=False, progress=None, batch_size=1_000_000, **kwargs):
+        """ Exports the DataFrame to a CSV file.
+
+        :param str path: Path for filename
+        :param bool virtual: If True, export virtual columns as well
+        :param bool selection: {selection1}
+        :param progress: {progress}
+        :param int batch_size: {chunk_size_export}
```

Comment:
```suggestion
        :param int chunk_size: {chunk_size_export}
```

Reply:
In vaex-core we use chunk_size everywhere, it;'s also consistent wording with arrow.